static/bidder-info/flatads.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://bid.rtbshark.com/api/rtbs/adx/rtb?x-net-id={{.PublisherID}}&x-net-token={{.TokenID}}"
+endpointCompression: gzip
+maintainer:
+  email: "adxbusiness@flat-ads.com"

```

Comment:
We've sent an email to this address. Please respond to the email with a message of "received" so we have confirmation that this address is active.

Reply:
Confirmed email id